tests/cypress/support/commands.ts

```diff
@@ -182,3 +185,14 @@ Cypress.Commands.add('deleteApplicationDeployment', (appNamespace, clusterName='
   cy.clickNavMenu(['Workloads', 'Deployments']);
   cy.deleteAll({fleetCheck: false});
 });
+
+// Modify given application
+Cypress.Commands.add('modifyDeployedApplication', (appName, clusterName='local') => {
+  cypressLib.burgerMenuToggle();
+  cypressLib.accesMenu(clusterName);
+  cy.clickNavMenu(['Workloads', 'Deployments']);
+  // Modify deployment of given application
+  cy.get('#trigger').click({ force: true });
+  cy.get('.icon-plus').click();
+  cy.get('.icon.icon-chevron-up').click({ force: true });
```

Comment:
This locator does not work in 2.7 at least as it finds multiple ones.
At least there this other one does work: `cy.get('#trigger > .icon-chevron-up').click({ force: true });`

Reply:
okay, let me try with that and update the patch.